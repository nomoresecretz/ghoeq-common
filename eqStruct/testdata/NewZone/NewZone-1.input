enp6egAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAG5la3R1bG9zAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAATmVrdHVsb3MgRm9yZXN0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD/UFBQUFpaWlpGRkZGAAAAIEEAACBBAAAgQQAAIEEAAHpDAAB6QwAAekMAAHpDzczMPgEKCgUFAgMBAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAIAAAAAAAAAQD8AIJbEAICLwwAAoMAAAP5DAAA9wwAASEMAgIlDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=