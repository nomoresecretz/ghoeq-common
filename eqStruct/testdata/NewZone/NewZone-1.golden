"name:  \"zzzz\"\nname_short:  \"nektulos\"\nname_long:  \"Nektulos Forest\"\ntype:  255\nfog_red:  80\nfog_green:  90\nfog_blue:  70\ngravity:  205\ntime_type:  1\nrain_chance:  10\nrain_duration:  2\nspecial_dates:  \"\\x00\\x00\\x00\\x00\"\nspecial_codes:  \"\\x00\\x00\\x00\\x00\"\nwater_music:  4\nmusic_normal_day:  2\ngraveyard_time:  256\n"